docs/src/content/docs/commands/BFADD.md

```diff
@@ -48,7 +51,7 @@ In this example, the item "apple" is added to the Bloom Filter named `mybloomfil
 ### Adding an Existing Item to a Bloom Filter
 
 ```plaintext
```

Comment:
@vs4vijay changes look good to me, minor comment please change `plaintext` to `bash` I believe that'd be in parity with other docs.